be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/RollCallValidator.scala

```diff
@@ -91,12 +108,19 @@ case object RollCallValidator extends MessageDataContentValidator with EventVali
           EVENT_HASH_PREFIX, laoId.toString, data.closes.toString, data.closed_at.toString
         )
 
+        val sender: PublicKey = message.sender
+        val channel: Channel = rpcMessage.getParamsChannel
+
         if (!validateTimestampStaleness(data.closed_at)) {
           Right(validationError(s"stale 'closed_at' timestamp (${data.closed_at})"))
         } else if (data.attendees.size != data.attendees.toSet.size) {
           Right(validationError("duplicate attendees keys"))
         } else if (expectedRollCallId != data.update_id) {
           Right(validationError("unexpected id 'update_id'"))
+        } else if (!validateOwner(sender, channel)){
+          Right(validationError(s"invalid sender $sender"))
+        } else if (!validateChannelType(ObjectType.LAO, channel)) {
```

Comment:
A comment here maybe :+1:

Reply:
I added one at the beginning of the class